vue/components/atoms/error-alert/error-alert.vue

```diff
@@ -0,0 +1,298 @@
+                                v-bind:width="15"
+                                v-bind:height="15"
+                                v-bind:icon="'spam'"
+                            />
+                        </div>
+                        <div class="header-text">
+                            {{ message }} @ {{ file }}:{{ sourceLocation }}
+                        </div>
+                    </div>
+                    <div class="right">
+                        <icon
+                            class="arrow"
+                            v-bind:color="'#ffffff'"
+                            v-bind:stroke-width="2"
+                            v-bind:width="15"
+                            v-bind:height="15"
+                            v-bind:icon="'chevron-up'"
+                        />
+                        <icon
+                            class="close"
+                            v-bind:color="'#ffffff'"
+                            v-bind:stroke-width="2"
+                            v-bind:width="15"
+                            v-bind:height="15"
+                            v-bind:icon="'close'"
+                            v-on:click.stop="onCloseClick"
+                        />
+                    </div>
+                </div>
+                <transition name="fade">
+                    <div class="error-stack" v-show="!isCollapsed">
+                        <p
+                            class="stack-line"
+                            v-for="(stackLine, index) of stackLines"
+                            v-bind:key="index"
+                        >
+                            {{ stackLine }}
+                        </p>
+                    </div>
+                </transition>
+            </div>
+        </div>
+    </transition>
+</template>
+
+<style lang="scss" scoped>
+.error-alert {
+    color: #ffffff;
+    display: block;
+    font-family: "consolas", monospace;
+    font-size: 12px;
+    opacity: 1;
+    transition: opacity 1s ease-in-out;
+}
+
+body.round .error-alert {
+    border-radius: 4px 4px 4px 4px;
+}
+
+.error-alert > .error-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.error-alert.direction-top > .error-container {
+    display: flex;
+    flex-direction: column-reverse;
+}
+
+.error-alert > .error-container > .error-header {
+    background-color: #e96760;
+    cursor: pointer;
+    display: flex;
+    font-weight: bold;
+    padding: 8px 8px 8px 8px;
+}
+
+.error-alert > .error-container > .error-header > .left {
+    display: inline-flex;
+    justify-content: space-around;
+    min-width: 150px;
+    width: 100%;
+}
+
+.error-alert > .error-container > .error-header > .left > .header-icon {
+    align-items: center;
+    display: inline-flex;
+    max-height: 15px;
+    padding-right: 4px;
+}
+
+.error-alert > .error-container > .error-header > .left > .header-text {
+    -webkit-box-orient: vertical;
+    display: inline-block;
+    display: -webkit-box;

```

Comment:
why this?

Reply:
it's a CSS method to do text overflow when text surpasses 2 lines (see `.error-alert.collapsed > .error-container > .error-header > .left > .header-text` selector)

Reply:
removed duplicated `display`

Reply:
This is what I could find:
![image](https://user-images.githubusercontent.com/24736423/157421977-68891728-788b-4c9f-9f71-1bbd0244c683.png)

src: https://css-tricks.com/almanac/properties/l/line-clamp/#aa-browser-support

Reply:
The mobile/tablet versions are relatively recent ones and using webkit is always risky: